udp/udp_linux.go

```diff
@@ -46,8 +47,15 @@ const (
 type _SK_MEMINFO [_SK_MEMINFO_VARS]uint32
 
 func NewListener(l *logrus.Logger, ip string, port int, multi bool, batch int) (*Conn, error) {
+	lip := net.ParseIP(ip)
+	lipV4, isV4 := isIPV4(lip)
```

Comment:
This exists in `package cidr` but isn't accessible here. Same for `isZeros()` within.

Reply:
Ah, makes sense. Forgot to run tests after rebasing.
Fixed now.